app/models/node.rb

```diff
@@ -850,6 +850,15 @@ def add_tag(tagname, user)
             # add sub-tags:
             subtags = {}
             subtags['pm'] = 'particulate-matter'
+            subtags['pm'] = 'air-quality'
```

Comment:
this key is what is causing unit test to fail

Reply:
Yes, great catch!!

```suggestion
```

Here, i've made this change as a "suggestion" which @aliciapaz can accept!